From 828d96c2bdd5b4df26bc05a26983fb4c70305eff Mon Sep 17 00:00:00 2001
From: Aswin M <aswin_m@comcast.com>
Date: Mon, 27 Jun 2022 06:41:02 +0000
Subject: [PATCH] HX44X-149 : Bring in Voice Support

Reason for change: Transparent background for AVS overlay screens.
Test Procedure: Enable necessary configurations and build with avs-sdk-oobe-screens.
Risks: Low
License: Inherited
Upstream-Status: Pending
Source: RDKM

Signed-off-by: Aswin M <Aswin_M@comcast.com
---
 modules/GUI/js/src/components/sampleHome.css | 3 ++-
 modules/GUI/js/src/main.css                  | 4 ++--
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/modules/GUI/js/src/components/sampleHome.css b/modules/GUI/js/src/components/sampleHome.css
index 1c67aaf..006ed5c 100644
--- a/modules/GUI/js/src/components/sampleHome.css
+++ b/modules/GUI/js/src/components/sampleHome.css
@@ -4,6 +4,7 @@
     flex-direction: column;
     align-items: center;
     justify-content: center;
+    opacity: 0.0;
     height: 100%;
 }
 
@@ -39,4 +40,4 @@
     color: lightgray;
     padding-top: 2vh;
     text-align: center;
-}
\ No newline at end of file
+}
diff --git a/modules/GUI/js/src/main.css b/modules/GUI/js/src/main.css
index 1617431..b9de33e 100644
--- a/modules/GUI/js/src/main.css
+++ b/modules/GUI/js/src/main.css
@@ -8,7 +8,7 @@
 
 body, html {
     overflow: hidden;
-    background-color: black
+    background: rgba(0, 0, 0, 0.0)
 }
 
 * {
@@ -31,5 +31,5 @@ body, html {
 
     flex-direction: column;
     justify-content: flex-end;
-    background-color:  var(--display-color);
+    background: rgba(2, 14, 19, 0.0);
 }
-- 
2.17.1

